docs/time/index.rst

```diff
@@ -1177,19 +1296,29 @@ dynamical models). An example using the JPL ephemerides is:
     >>> (ltt_bary_jpl - ltt_bary).to(u.ms) # doctest: +REMOTE_DATA +IGNORE_OUTPUT
     <Quantity [-0.00132325, -0.00132861] ms>
 
-The difference between the builtin ephemerides and the JPL ephemerides is normally
-of the order of 1/100th of a millisecond, so the builtin ephemerides should be suitable
-for most purposes. For more details about what ephemerides are available,
-including the requirements for using JPL ephemerides, see :ref:`astropy-coordinates-solarsystem`.
+The difference between the built-in ephemerides and the JPL ephemerides is
+normally of the order of 1/100th of a millisecond, so the built-in ephemerides
+should be suitable for most purposes. For more details about what ephemerides
+are available, including the requirements for using JPL ephemerides, see
+:ref:`astropy-coordinates-solarsystem`.
+
+.. EXAMPLE END
 
-Interaction with Time-like Quantities
+Interaction with Time-Like Quantities
 -------------------------------------
 
-Where possible, |Quantity| objects with units of time are treated as TimeDelta
+Where possible, |Quantity| objects with units of time are treated as |TimeDelta|
 objects with undefined scale (though necessarily with lower precision). They
 can also be used as input in constructing |Time| and |TimeDelta| objects, and
 |TimeDelta| objects can be converted to |Quantity| objects of arbitrary units
-of time.  Usage is most easily illustrated by examples::
+of time. Usage is best illustrated by example.
```

Comment:
Maybe delete "Usage is ..." since there is a nice new "examples" section following right after